docs/manage/ml/train-model.md

```diff
@@ -10,37 +10,26 @@ description: "Train an image classification model on labeled image data."
 # SME: Aaron Casas
 ---
 
-You can label or add bounding boxes to [images collected](/services/data/configure-data-capture/) by robots and use the annotated data to train a **Single Label Classification Model**, **Multi Label Classification Model** or **Object Detection Model** within Viam.
-
-{{<youtube embed_url="https://www.youtube-nocookie.com/embed/CP14LR0Pq64">}}
+You can label or add bounding boxes to [images collected](/services/data/configure-data-capture/) by robots, add them to a dataset, and use the annotated data to train a **Single Label Classification Model**, **Multi Label Classification Model** or **Object Detection Model** within Viam.
 
 When training machine learning models, it is important to supply a variety of different data about the subject.
 In the case of image classification, it is important to provide images of the object being identified in different situations, such as from different angles or in different lighting situations.
 The more varied the provided data set, the more accurate the resulting model becomes.
 
 ## Train a model
 
-After [annotating your images](/manage/data/label/), click on the **TRAIN MODEL** button in the top right corner.
-
-![Train model button](/manage/ml/train-model.png)
+After [creating a dataset](/manage/data/dataset/), navigate to the **DATA** tab and the **DATASETS** subtab.
+Then click on the dataset you want to train a model from and click on the **Train model** button on your dataset's page.
 
-A **Training** side menu opens.
-The model will train on all images that are part of the current filter.
-
-{{< alert title="Info" color="info" >}}
-Filtered datasets are views and not materialized.
-That means the data you are viewing may change as you label and train on the dataset.
-If the underlying data matching the filter changes because data is deleted or more data is added, the dataset will also change.
-{{< /alert >}}
+On the **Train a model** menu:
 
-1. Select **New Model**.
-2. Specify a **Model Name**.
-3. Select a **Model Type** and one or more labels to train on:
+1. Enter a name for your new model.
+1. Select a **Model Type** and one or more labels to train on:
    - **Single Label Classification**: The resulting model predicts one of the selected labels or `UNKNOWN` per image.
      If you are only using one label, ensure that the dataset you are training on also contains unlabeled images.
    - **Multi Label Classification**: The resulting model predicts one or more of the selected labels per image.
    - **Object Detection**: The resulting model predicts either no detected objects or any number of object labels alongside their locations per image.
-4. Click **TRAIN MODEL**
+1. Click **TRAIN MODEL**
 
 ![Train model menu](/manage/ml/train-model-menu.png)
 
```

Comment:
In the section below:

> Once the model has finished training, it becomes visible in the **Models** section of the page.

A notable addition is that the model will have a link to the dataset it was trained on. You can see this in https://app.viam.dev/data/models if you've trained a model on a dataset, e.g.:
<img width="1390" alt="image" src="https://github.com/viamrobotics/docs/assets/3794748/bf05d377-9221-43e1-91ab-9e149e437d5e">

It's worth noting that the dataset can have images added or removed to it since that model was trained, and it's not currently versioned.

Reply:
I can't confirm that the Dataset is visible Alexa. Is this still in progress?
<img width="815" alt="Screenshot 2023-11-21 at 10 42 51" src="https://github.com/viamrobotics/docs/assets/5212232/c44d8da6-ab76-462d-9488-c848e9b4052e">

Reply:
This will be fixed now if you retrain a new model on a dataset.

Reply:
This would be the only blocking update before merging these docs; I think my other one regarding the subtle difference with 'Train' in the screenshots can be updated after.